ports/stm32/sdram.c

```diff
@@ -39,6 +39,11 @@
 #define SDRAM_START_ADDRESS 0xD0000000
 #endif
 
+// Provides the MPU_REGION_SIZE_X value when passed the size of region in bytes
+#define MPU_REGION_SIZE(m) ((m-1)/((m-1)%255+1) / 255%255*8 + 7-86/((m-1)%255+12) - 1)
```

Comment:
It's probably worth stating the restrictions on `m`, looks like it must be a power of 2, up to 2**2039.

Reply:
I figured that limit is so much higher than the supported range of the MPU settings that it wasn't too relevant. Commenting and/or limiting it to the settings range actually available for the MPU is more likely useful though.